src/Uno.UI/ts/WindowManager.ts

```diff
@@ -1391,11 +1391,13 @@ namespace Uno.UI {
 			const elementStyle = element.style;
 			const elementClasses = element.className;
 			const originalStyleCssText = elementStyle.cssText;
+			const unconstrainedStyleCssText = this.createUnconstrainedStyle(elementStyle, maxWidth, maxHeight);
+
 			let parentElement: HTMLElement = null;
 			let parentElementWidthHeight: { width: string, height: string } = null;
 			let unconnectedRoot: HTMLElement = null;
 
-			let cleanupUnconnectedRoot = function (owner: HTMLDivElement) {
+			let cleanupUnconnectedRoot = (owner: HTMLDivElement) => {
```

Comment:
Codacy found an issue: [Identifier 'cleanupUnconnectedRoot' is never reassigned; use 'const' instead of 'let'.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=6420005)